### PR TITLE
Adds decorative bones and bone loot pile

### PR DESCRIPTION
**Pull request by BillyBangles**
Credit to schnayy for most of these sprites. Just some bones for mapping/POI decoration, as well as a new bone loot pile, for <del>more morally questionable shit for the explorers to do</del> all of your corpse looting needs.

alt text

### CODE REVIEW

File: code/game/objects/structures/loot_piles.dm [hunk 1]
```diff
@@ -522,3 +522,51 @@ Loot piles can be depleted, if loot_depleted is turned on.  Note that players wh
...
+    common_loot = list(
+        /obj/item/weapon/bone,
+        /obj/item/weapon/bone/skull,
+        /obj/item/weapon/bone/skull/tajaran,
+        /obj/item/weapon/bone/skull/unathi,
+        /obj/item/weapon/bone/skull/unknown,
+        /obj/item/weapon/bone/leg,
+        /obj/item/weapon/bone/arm,
+        /obj/item/weapon/bone/ribs,
+        /obj/item/weapon/arrow
```
Comment (Anewbe) [inline code/game/objects/structures/loot_piles.dm]: Not really sure why this is in this list? That item, as far as I can tell, only exists to be thrown by the crossbow, and even then I'm not sure it works for that.

Comment (BillyBangles) [reply code/game/objects/structures/loot_piles.dm]: My mindset was along the lines of "This is something that might've killed the person all these bones belong to." I was also thinking of putting a bullet casing in there as well or something but it slipped my mind. I could remove it if you feel it's too out of place.

Comment (BillyBangles) [reply code/game/objects/structures/loot_piles.dm]: It does work for the crossbow, by the by. I did test that, and it works.

Comment (Anewbe) [reply code/game/objects/structures/loot_piles.dm]: Huh. Good to know. I'd still remove this. There are knives in the uncommon loot, that's... probably fine?
And a lot of what'll be killing people, ICly, is the climate.

Comment (BillyBangles) [reply code/game/objects/structures/loot_piles.dm]: Fair enough. Can do.

Comment (Anewbe) [reply code/game/objects/structures/loot_piles.dm]: Thank you.

**Comment by Anewbe** [comment]
Compiles, etc?

**Comment by BillyBangles** [comment]
yup, sure does

**Pull request merged**